app/javascript/lib/study-overview.js

```diff
@@ -0,0 +1,338 @@
+/**
+* @fileoverview Default view of Explore tab in Study Overview
+*
+* Shows "Clusters" and sometimes "Genomes", etc.
+*/
+
+/**
+ * Render cluster code
+ *
+ * TODO: Move this to render-cluster.js, or some such
```

Comment:
ticket?  or rename from TODO to 'perhaps'?

Reply:
Thanks for the catch!  Ticketed, and added to code comment, in b24e07f.